doc/source/users/environment_variable_catalog.rst

```diff
@@ -0,0 +1,352 @@
+.. _environment-variable-catalog:
+
+============================
+Environment Variable Catalog
+============================
+
+This is the catalog of the available environment variables that can be used in
+Sardana.
+
+.. toctree::
+    :maxdepth: 3
+
+    environment_variable_catalog
```

Comment:
This causes `circular toctree references` warnings.

Reply:
I'm trying to find a solution for this case ...